docs/one-click-integrations.md

```diff
@@ -22,15 +36,25 @@ Pulse integrates directly with GitHub to receive data about changes and deployme
 
         Because of this, the integration discards the first SemVer tag in the repository since there is no previous tag to compare with.
 
-To set up the GitHub integration:
+    -   Make sure that you're creating a Git tag on your repositories for each successful deployment to production, or whenever you make a new release available to any user of your application:
```

Comment:
This is useful, we didn't have this before but it makes clear what are the exact requirements that Pulse needs to correctly detect the deployments. :+1:

Reply:
It was bellow in the first step, but since the steps now are common, I moved it here.